datadreamer/pipelines/generate_dataset_from_scratch.py

```diff
@@ -227,6 +228,12 @@ def parse_args():
         help="Path to the configuration file",
     )
 
+    parser.add_argument(
+        "--dataset_plugin",
+        type=str,
+        help="Dataset plugin for luxonis-dataset",
```

Comment:
```suggestion
        help="LuxonisDataset plugin for the luxonis-dataset format",
```
Might be a bit more descriptive